test/access-token.test.js

```diff
@@ -238,16 +290,19 @@ describe('AccessToken', function() {
     });
 
     function mockRequest(opts) {
-      return extend(
-        {
+      return extend({
           method: 'GET',
           url: '/a-test-path',
           headers: {},
           _params: {},
 
           // express helpers
-          param: function(name) { return this._params[name]; },
-          header: function(name) { return this.headers[name]; }
+          param: function(name) {
+            return this._params[name];
+          },
+          header: function(name) {
+            return this.headers[name];
+          }
```

Comment:
Unrelated formatting/space changes, please revert. Please check also all modified lines below, the same comment applies there too.